### ISSUE TITLE
Watch for Tesla App Installation Issue

### ISSUE THREAD

**Issue by cammibailey** [Kemmey/Kemmey-TeslaWatch-Public#153]
Installed on Apple Watch 5, entered Tesla Login,  app will not login. It says “could not acquire token:Request failed. Response code 401 Message: unauthorized.

Was my husbands watch, wiped it clean and set up as mine. He had same app on it, but we purchased it Again new after wipe because he liked it and recommended it for me. 

Frustrated can’t get it to login.

**Comment by Kemmey** [comment]
Erfor code 401 means either wrong password or account locked. Your account can easily get locked out, as it will lock after just two tries with wrong password. 

Please try to login on Tesla.com and verify your account still works properly. 

Another note - the app supports family sharing, so you can go ahead and request a refund for the second purchase and setup family sharing and get the second copy for free :-D

**Comment by cammibailey** [comment]
Yes, that was the problem. I was locked out.  I reset and now it’s fixed.  Thanks for the tip on family sharing.

Thank you!
